src/debugger/android/androidPlatform.ts

```diff
@@ -19,13 +20,56 @@ export class AndroidPlatform implements IAppPlatform {
 
     public enableJSDebuggingMode(runOptions: IRunOptions): Q.Promise<void> {
```

Comment:
The method name doesn't match what the method does any more. We should either rename the method, or move the logic to different method.